<!DOCTYPE HTML>
<!--
	Massively by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>gavin c. woodruff, ph.d. - research interests</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-preload">

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Header -->
					<header id="header">
						<a href="index.html"><font size="12">Gavin C. Woodruff, Ph.D.</font></a>
						<a href="index.html"></a>
					</header>

				<!-- Nav -->
					<nav id="nav">
						<ul class="links">
							<li><a href="index.html">home</a></li>
							<li class="active"><a href="research_interests.html">research interests</a></li>
							<li><a href="publications.html">publications</a></li>
							<li><a href="cv.html">cv</a></li>
							<li><a href="little_thoughts.html">little thoughts</a></li>
						</ul>
					</nav>

				<!-- Main -->
					<div id="main">

					<div id="main">
							<!-- Image -->
								<section>
									<header>
										<h1></h1>
										<p>What are the causes of morphological change?</p>
									</header>
							<p><span class="image left"><img src="images/phylogeny.png" alt="" /></span> What is the genetic, cellular, developmental, and evolutionary basis of phenotypic diversity? This question can be addressed from many perspectives— molecular geneticists, developmental biologists, population geneticists and ecologists have typically tackled these problems using very different approaches. However, as phenotype construction has multiple causes, a satisfying understanding of phenotypic diversity ultimately requires the integration of multiple points of view. I am currently using the nematode <a href="https://www.nature.com/articles/s41467-018-05712-5"><i>Caenorhabditis inopinata</i></a> to connect functional genetics with evolution and ecology to understand the causes of phenotypic diversity. <a href="https://doi.org/10.1002/evl3.67"><i>C. inopinata</i> can grow to be nearly twice as long as its close relatives</a>, which include the highly-studied model organism, <a href="http://www.wormbook.org"><i>C. elegans</i></a>. Furthermore, it thrives in the fresh <i>Ficus septica</i> figs of Okinawa and is associtead with its pollinating wasps; <a href="http://www.figweb.org">figs and fig wasps together represent a classic system in evolution and ecology</a>. <i>C. inopinata</i> is then well-positioned to connect multiple disciplines that aim to understand the bases of phenotypic variation. To this end, I am continuing to harness genomic, evolutionary, and developmental genetic approaches to address these questions using this system.</p>
								</section>
								<section>
									<header>
										<h1></h1>
										<p>What genes underlie developmental and morphological divergence?</p>
									</header>
							<p><span class="image right"><img src="images/dev_inopinata-01.png" alt="" /></span> <i>C. inopinata</i> is large and slow-developing, and some of the earliest and most widely studied mutants in <i>C. elegans</i> affect body size and the timing of developmental events. Could changes in these genes underlie heterochrony and body size evolution? In addition to interrogating these obvious candidate genes, I am using genome-wide phylogenetic and molecular evolution approaches to discover potential genetic drivers of developmental change. Because of the context of the <i>C. elegans</i> system, these candidates can then be directly tested with conventional developmental genetic experiments. </p>
								</section>
								<section>
									<header>
										<h1></h1>
										<p>What ecological factors drive evolutionary change?</p>
									</header>
							<p><span class="image left"><img src="images/wasps.png" alt="" /></span> The environment propels adaptation. <i>C. elegans</i> and most <i> Caenorhabditis</i> species thrive on rotting plants, whereas <i>C. inopinata</i> grows in fresh figs. Clearly its radically different environment has influenced its phenotypic divergence, but how? <a href="https://doi.org/10.1186/s12898-018-0182-z">We have recently defined the natural history of this species</a> to generate hypotheses regarding the ecological drivers of divergence. For instance, <i>C. inopinata</i> shows a dispersal preference for pollinating wasps over parasititc wasps— how does fig wasp diversity impact <i>C. inopinata</i> fitness? Future field work interrogating patterns of microbial, nematode, and fig wasp diversity will be connected with laboratory experiments in order to understand the causes of species distributions.</p>
								</section>
								<section>
									<header>
										<h1></h1>
										<p>What are the life history consequences of body size change?</p>
									</header>
							<p><span class="image right"><img src="images/life_history.png" alt="" /></span> Organisms appear to adopt life history strategies where patterns of reproduction, growth, and body size covary. <i>C. inopinata</i>	is huge in size and grows much slower than its close relatives. Are these traits meaningfully connected in <i>C. inopinata</i>, and can this system be used to understand how life history traits evolve? <a href="https://doi.org/10.1186/s12862-019-1388-1">We have recently described the life history traits of this species</a> and showed that lifespan is not extended in this large, slow-developing species. We are eager to implement both quantitative genetic and candidate gene/reverse genetic approaches to understand the relationship among life history traits in this group.</p>
								</section>
								<section>
									<header>
										<h1></h1>
										<p>What are the consequences of gene loss, and can gene loss be predicted?</p>
									</header>
							<p><span class="image left"><img src="images/transcripts.png" alt="" /></span>Gene losses have been known to underlie adaptive change. At the same time, gene knockdown often has no obvious functional consequence. How common is genes loss, and what kinds of genes are dispensible? And further, has <i>C. inopinata</i> lost any genes that may underlie its unique morphology and ecological context? I am interrogating patterns of gene loss in nematodes and connecting them with <a href="http://www.modencode.org/publications/about/index.shtml">pre-existing transcriptomic data sets</a> and <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-32">nematode-specific gene ontologies</a> to understand the causes and consequences of gene loss.
</p>
								</section>
								<section>
									<header>
										<h1></h1>
										<p>How is genomic diversity structured within genomes and among populations? </p>
									</header>
							<p><span class="image right"><img src="images/pi.png" alt="" /></span> Interspecies divergence begins as intraspecies variation. What is the pattern, structure, and history of such variation? I am currently characterizing <i>C. inopinata</i> genomic variation to understand patterns of diversity across fig, tree, and island populations. I will use these genotypes to estimate population-level diversity, determine the organization of diversity both within genomes and across geographic space, and infer the demographic histories and evolutionary forces that shape populations and genomes. </p>
								</section>
								<section>
									<header>
										<h1></h1>
										<p>How does gene family evolution influence morphological and ecological divergence and vice-versa?</p>
									</header>
							<p><span class="image left"><img src="images/spec_phy.png" alt="" /></span> Gene duplication and neofunctionalization is a common engine of adaptation in multiple systems. What gene families exhibit frequent expansion and contraction, and are there any ecological predictors of gene copy number variation? Furthermore, are there any gene families that have exhbited unique copy number evolution in the exceptional <i>C. inopinata</i>, and could these help to explain its unique biology? I am currently using genomic and phylogenetic comparative approaches to understand how the evolution of host-specificity impacts gene family change.
 </p>
								</section>								
								
								<section>
									<header>
										<h1></h1>
										<p>How does ecological divergence impact polyphenism?</p>
									</header>
							<p><span class="image right"><img src="images/dauer.PNG" alt="" /></span> Selection does not only mold populations across generations-- it also confers organisms with the ability to respond to environmental change within a generation. How does phenotypic plasicity evolve upon drastic ecological change? Many nematodes can respond to adverse environmental conditions by developing into a stress-resistant diapause stage. This stage has clearly diverged in morphological space in <i>C. inopinata</i>; this change is likely connected to its need to disperse on pollinating fig wasps. I am interested in implementing quantitative and reverse genetic approaches in order to understand the evolution of plasticity. </p>
								</section>
								<section>
									<header>
										<h1></h1>
										<p>How and why do genomic structures and repetitive elements evolve?</p>
									</header>
							<p><span class="image left"><img src="images/repeats.png" alt="" /></span> Repetitive elements can dominate eukaryote genomes. What favors and constrains their expansion among lineages and within genomes? Why do repetitive element distributions vary among repeat classes? The <i>C. inopinata</i> genome reveals rampant repetitive element expansion and a genomic structure of repeats that is drastically divergent from its close relatives. I am using comparative genomic approaches to understand how repetitive elements evolve. </p>
								</section>
								<section>
									<header>
										<h1></h1>
										<p>Other crazy stuff I am interested in</p>
									</header>
							<p><span class="image left"><img src="images/mating_damage.PNG" alt="" />
									<div class="row">
											<ul>
												<li>Reproductive isolation and the evolution of hermaphroditism.</li>
												<li>Sexual conflict and reproductive trade-offs (mating-induced damage in figure).</li>
												<li>The evolution of phoresis and invertebrate-carrier sensing.</li>
												<li>Neglected and recently-discovered <i>Caenorhabditis</i> species.</li>
											</ul>
								</section>
					</div>

				<!-- Footer -->
					<footer id="footer">
						<section class="split contact">
							<section>
								<h3>Email</h3>
								<p>gavincw(at)uoregon(dot)edu</p>
							</section>
							<section>
								<h3>Tweet Machine</h3>
								<p><a href="https://twitter.com/weird_worms">@weird_worms</a></p>
							</section>
						</section>
					</footer>

				<!-- Copyright -->
					<div id="copyright">
						<ul><li>&copy; 2018</li><li>Design: <a href="https://html5up.net">HTML5 UP</a></li></ul>
					</div>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
